recipes/qt/6.x.x/conanfile.py

```diff
@@ -1494,6 +1494,8 @@ def _create_plugin(pluginname, libname, plugintype, requires):
                 # https://github.com/qt/qtbase/blob/v6.6.1/src/network/CMakeLists.txt#L216-L221
                 # qtcore requires "_OBJC_CLASS_$_NSApplication" and more, which are in "Cocoa" framework
                 self.cpp_info.components["qtCore"].frameworks.append("Cocoa")
+                # https://github.com/qt/qtbase/blob/v6.8.3/src/corelib/CMakeLists.txt#L712-L717
+                self.cpp_info.components["qtCore"].frameworks.append("UniformTypeIdentifiers")
```

Comment:
This looks like it was added as a requirement for all versions >= 6.8.0
https://github.com/qt/qtbase/commit/2a2a30155719d9175f92467a42e0deac5aad3767

may be worth appending this conditionally on version being >= 6.8.0?